pkg/operator/baremetal_pod.go

```diff
@@ -173,15 +173,12 @@ func newMetal3PodTemplateSpec(config *OperatorConfig, baremetalProvisioningConfi
 			},
 		},
 		Spec: corev1.PodSpec{
-			Volumes:           volumes,
-			InitContainers:    initContainers,
-			Containers:        containers,
-			HostNetwork:       true,
-			PriorityClassName: "system-node-critical",
-			NodeSelector:      map[string]string{"node-role.kubernetes.io/master": ""},
-			SecurityContext: &corev1.PodSecurityContext{
-				RunAsNonRoot: pointer.BoolPtr(false),
-			},
```

Comment:
Oh wait, I missed this bit - why did you remove this?

Reply:
I thought this was delegated to whatever level of isolation give the SCC in use now, so no need to explicitly set this?

Reply:
I haven't looked into this part at all, but I guess I understood that RunAsNonRoot: true is the default when we remove the run-level ... so we'd still explicitly need to explicitly set it where we want RunAsNonRoot: false

Reply:
I was reading `RunAsNonRoot: true` :/. I'll update